### PR TITLE
Gradle: Improve the code comment about applying system properties

### PR DESCRIPTION
**Pull request by sschuberth**
Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

File: analyzer/src/main/kotlin/managers/Gradle.kt [hunk 1]
```diff
@@ -126,9 +126,12 @@ class Gradle(
     override fun resolveDependencies(definitionFile: File): ProjectAnalyzerResult? {
         val gradleSystemProperties = mutableListOf<Pair<String, String>>()
 
-        // Usually, the Gradle wrapper handles applying system properties from Gradle properties. But as we directly use
-        // the tooling API, we need to manually load Gradle properties and apply any system properties. Limit the lookup
-        // to the current user's Gradle properties file for now.
+        // Usually, the Gradle wrapper's Java code handles applying of system properties defined in a Gradle properties
```
Comment (fviernau) [inline analyzer/src/main/kotlin/managers/Gradle.kt:129]: "handles applying of": should it be either "handles the application of" or "handles appyling"?

File: analyzer/src/main/kotlin/managers/Gradle.kt [hunk 2]
```diff
@@ -126,9 +126,12 @@ class Gradle(
...
+        // file. But as we use the Gradle Tooling API instead of the wrapper to start the build, we need to manually
+        // load any system properties from a Gradle properties file and set them in the process that uses the Tooling
+        // API. A typical use-case for this is to apply proxy settings so that the Gradle distribution used by the build
```
Comment (fviernau) [inline analyzer/src/main/kotlin/managers/Gradle.kt:132]: "use-case": maybe "use case"?

Comment (sschuberth) [reply analyzer/src/main/kotlin/managers/Gradle.kt:132]: Done.

File: analyzer/src/main/kotlin/managers/Gradle.kt (hunk 1 above)
Comment (sschuberth) [reply analyzer/src/main/kotlin/managers/Gradle.kt]: Done.

**Pull request merged**